src/params/types.ts

```diff
@@ -159,6 +164,7 @@ type ParamValueType = "string" | "list" | "boolean" | "int" | "float" | "secret"
 type ParamInput<T> =
   | { text: TextInput<T> }
   | { select: SelectInput<T> }
+  | { selectList: SelectListInput }
```

Comment:
I thought this was called MultiSelect in the spec since that name has been common since the days of jQuery.